src/screens/LoginScreen.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useState } from 'react';
+import { StatusBar } from 'expo-status-bar';
+import {
+  StyleSheet,
+  Text,
+  View,
+  SafeAreaView,
+  KeyboardAvoidingView,
+  Platform,
+  TouchableWithoutFeedback,
+  Keyboard
+} from 'react-native';
+import LabeledInput from '../components/LabeledInput';
+import Button from '../components/Button';
+import { Formik } from 'formik';
+import { Colors, Fonts } from '../global-styles';
+import { platformValue } from '../util/platform';
+import Alert, { AlertProps } from '../components/Alert';
+import loginAPI from '../util/API';
```

Comment:
Should be `import API from '../util/API';`, it's usually good practice for default exports to match the name of the file.